### PR TITLE
More logging for riff image relocate

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/projectriff/riff/issues/851

### CODE REVIEW

File: cmd/commands/image.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func ImageRelocate(c *core.ImageClient) *cobra.Command {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := (*c).RelocateImages(options)
+			err := (*c).RelocateImages(options, cmd.OutOrStdout())
```
Comment (ericbottard) [inline cmd/commands/image.go]: Would be better to inject the Writer into `imageClient` in `wiring.go` IMO, to be consistent with other signatures

Comment (glyn) [reply cmd/commands/image.go]: Yes, of course, good idea. I'll look into that now.

Comment (glyn) [reply cmd/commands/image.go]: Fixed.

**Comment by ericbottard** [comment]
Merged as 6c5b7469

**Pull request closed without merging**